scripts/station_csv_to_json.py

```diff
@@ -18,6 +29,10 @@
 
     WEATHER_STATIONS = []
 
+    ECODIVISIONS = geopandas.read_file('app/data/ERC_ECODIV_polygon/ERC_ECODIV_polygon.shp')
```

Comment:
Can we move the shapefiles to a different folder please? It doesn't have to deploy as part of the app - unless you need it this way for local development in docker?